### PR TITLE
#161382339 Hound <> Github integration

### PR DESCRIPTION
**Pull request by cdvx**
#### What does this PR do?
Setup and customize Hound-Github integration

#### Description of Task to be completed?
Makes sure ESlint config and hound are configured to check for style violations

#### How should this be manually tested?
Raise a PR and Hound will make a review based on es6 coding style conventions.

#### Any background context you want to provide?
To test this out, one has to download and install `nodejs` locally (on the development machine)
install yarn `brew install yarn`
clone the repository
`git clone https://github.com/andela/ah-frontend-thor.git`

Then install the dependencies with
`yarn install` or `yarn`

#### What are the relevant pivotal tracker stories?
#161382339

### CODE REVIEW

File: .hound.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+fail_on_violations: true
+
+eslint:
+  version: 6
```
Comment (johnkegz) [inline .hound.yml]: @cdvx  add an empty line at the end of the file

Comment (cdvx) [reply .hound.yml]: @johnkegz , Working on it👍🏾

**Review by johnkegz — APPROVED** [review]

**Review by bozicschucky — APPROVED** [review]

File: .eslintrc [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+        "indent": [2, "tab"],
+        "react/jsx-indent": [2, "tab"],
+        "curly": [2, "multi-line"]
+    },
+    "parserOptions": {
+        "sourceType": "module"
+    },
+    "jsx-a11y/label-has-for": "off"
+    
+}
```
Comment (shakirandagire) [inline .eslintrc]: Could you add a new line at the end of the file

Comment (cdvx) [reply .eslintrc]: @shakirandagire Working on it

**Comment by emukungu** [comment]
Both eslint and hound files added. LGTM

**Pull request merged**